### PR TITLE
Allow multiple class names on buttons

### PR DESCRIPTION
**Pull request by krvajal**
This pull request allows the use of multiple class names when passing the `className` options to the button. The different classes should be separated by spaces.

### CODE REVIEW

**Comment by mbarwick83** [comment]
Is this getting pushed?

**Comment by pbarbiero** [comment]
`+         buttonEl.classList.add(className);`

Shouldnt that be

`+         buttonEl.classList.add(className_);`

**Comment by lionralfs** [comment]
Thanks for the PR, @krvajal 

I went ahead and added the possibility to also pass an array of classNames, as discussed in #743,
as well as some tests.

Looks good @t4t5?

File: src/modules/init/buttons.ts [hunk 1]
```diff
@@ -33,7 +33,17 @@ const getButton = (namespace: string, {
...
   if (className) {
-    buttonEl.classList.add(className);
+    if (typeof className === 'string') {
+      className.split(' ')
+        .filter(name => name.length > 0)
+        .forEach(name => {
+          buttonEl.classList.add(name);
+        });
+    } else if (Array.isArray(className)) {
+      className.forEach(name => {
```
Comment (krvajal) [inline src/modules/init/buttons.ts]: Please check the length of the name here, and also DRY when adding the class names. In both cases, we get an array from the input, so you could instead do this
```
const classNameArray = Array.isArray(className)? className: className.split(' ')
classNameArray.forEach(name => {
buttonEl.classList.add(name);
});
```
Makes sense?

Comment (lionralfs) [reply src/modules/init/buttons.ts]: Sure, great idea.
Do you agree with this?
```js
const classNameArray = Array.isArray(className) ? className : className.split(' ');
classNameArray
  .filter(name => name.length > 0)
  .forEach(name => {
    buttonEl.classList.add(name);
  });
```

Comment (krvajal) [reply src/modules/init/buttons.ts]: The first line can be split into parts for clarity if you want but yes, that's the idea.

**Comment by krvajal** [comment]
@t4t5  are you considering adding this?

**Comment by bahirul** [comment]
please merge this ... we need this feature :+1:

**Comment by t4t5** [comment]
Looks great guys! Thanks for the work @krvajal @lionralfs!

**Pull request merged**